src/lib/components/EventTime/index.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * @fileoverview Element that renders a time in local timezone.
+ */
+
+import {html} from 'lit-element';
+import {BaseElement} from '../BaseElement';
+import './_styles.scss';
+
+const months = [
+  'Jan',
+  'Feb',
+  'Mar',
+  'Apr',
+  'May',
+  'Jun',
+  'Jul',
+  'Aug',
+  'Sep',
+  'Oct',
+  'Nov',
+  'Dec',
+];
+
+const connected = new Set();
+let timezoneCheckInterval = 0;
+let previous = 0;
+
+const checkTimezone = () => {
+  const now = new Date().getTimezoneOffset();
+  if (now !== previous) {
+    previous = now;
+    connected.forEach((node) => node.requestUpdate());
+  }
+};
+
+/**
+ * Renders an event's time in the user's local timezone, including optional duration.
+ *
+ * @extends {BaseElement}
+ * @final
+ */
+class EventTime extends BaseElement {
+  static get properties() {
+    return {
+      datetime: {type: String},
+      duration: {type: Number},
+    };
+  }
+
+  constructor() {
+    super();
+    this._date = null;
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+
+    if (connected.size === 0) {
+      // Every 10 minutes, check if we're in the same timezone.
+      timezoneCheckInterval = window.setInterval(checkTimezone, 10 * 60 * 1000);
```

Comment:
Why does an element need to do this on an interval as opposed to checking once when it's connected?

Reply:
If the user's browser remains open but they change timezone, we want to re-render the data.

Reply:
lol. ain't nobody gettin' on planes anymore.

but ok i understand now 👍